src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj

```diff
@@ -193,7 +193,9 @@
   <Target Name="AfterBuild">
     <ItemGroup>
       <ExtensionDefinition Include="PythonMigration_ViewExtensionDefinition.xml" />
+      <CustomFixers Include="$(SolutionDir)PythonMigrationViewExtension\MigrationAssistant\custom_fixers\*" />
     </ItemGroup>
     <Copy SourceFiles="@(ExtensionDefinition)" DestinationFolder="$(OutputPath)\viewExtensions\" />
+    <Copy SourceFiles="@(CustomFixers)" DestinationFolder="$(OutputPath)custom_fixers\" />
```

Comment:
Maybe let's use a more specific name, like `pythonMigrationFixers`? This way we can easily identify what this is when seeing it in the Dynamo output folder.